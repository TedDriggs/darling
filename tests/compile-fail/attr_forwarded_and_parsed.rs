#[derive(darling::FromDeriveInput)]
#[darling(attributes(example), forward_attrs(example))]
pub struct Example {
    ignored: String,
}

#[derive(darling::FromField)]
#[darling(attributes(example), forward_attrs(example))]
pub struct ExampleField {
    ignored: String,
}

fn main() {}
