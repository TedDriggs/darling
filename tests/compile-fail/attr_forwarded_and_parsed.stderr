error: attribute path `example` will not be forwarded because it is also listed in `attributes`
 --> tests/compile-fail/attr_forwarded_and_parsed.rs:2:46
  |
2 | #[darling(attributes(example), forward_attrs(example))]
  |                                              ^^^^^^^

error: attribute path `example` will not be forwarded because it is also listed in `attributes`
 --> tests/compile-fail/attr_forwarded_and_parsed.rs:8:46
  |
8 | #[darling(attributes(example), forward_attrs(example))]
  |                                              ^^^^^^^
